src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/DeliusApiClient.kt

```diff
@@ -156,4 +170,11 @@ class DeliusApiClient(@Qualifier("oauthDeliusApiClient") val deliusApiWebClient:
     .retrieve()
     .toBodilessEntity()
     .block() ?: error("Unexpected response while assigning delius role for user: $username")
+
+  private fun <T> processErrors(it: Throwable): Mono<T> = when {
```

Comment:
the other clients call this `coerce404ResponseToNull` and log the request uri. 
could possibly extract out a response utils module and share

Reply:
this could also be used by `getOffenderManager` above